src/vs/languages/css/common/parser/cssSymbols.ts

```diff
@@ -102,6 +102,7 @@ export class Symbol {
 export class ScopeBuilder implements nodes.IVisitor {
```

Comment:
@aeschli This will definitely make it simple.. In this case, I think it is enough to add all css variables to global scope and do not need to add to local scope. Adding a variable to two scopes is not straight and is not necessary if we want to show all css defined variables during intellisense.